packages/apps/purchase-page/pages/promotions-page/stage5.js

```diff
@@ -73,6 +73,15 @@ function Stage5() {
                   },
                 } = clientData;
 
+                if (
+                    window && paymentType !== 'PayPal' &&
```

Comment:
This is going to throw on the server.

Use 
```suggestion
                    typeof window !== 'undefined' && paymentType !== 'PayPal' &&
```